sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_blobstoragecsaio.py

```diff
@@ -68,7 +68,7 @@ def from_connection_string(
         :param container_name:
             The container name for the blobs.
         :type container_name: str
-        :param credential:
+        :keyword any credential:
```

Comment:
why any?

Reply:
this was to align with the typehint and typing in the class docstring for credential

Reply:
discussed with Anna and updated types -- resolving :)